src/NECompletion/NECVarTypeGuesser.class.st

```diff
@@ -18,6 +18,7 @@ Class {
 
 { #category : #private }
 NECVarTypeGuesser class >> classFrom: className [
+
```

Comment:
No flag here?

Reply:
I was planning on removing this one but forgot about it in the end. need to be pushed in the instance.